SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md

```diff
@@ -18,9 +17,17 @@ Before porting your project:
 * Open a terminal window and run the command `spatial update` to ensure your [spatial CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) installation is up to date. 
 
 ### Terms used in this guide
-`<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
-`<ProjectRoot>` - The directory containing your `<GameRoot>`.  
-`<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+* `<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
+* `<ProjectRoot>` - The directory containing your `<GameRoot>`.  
+* `<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+
+<%(Callout type="tip" message="
+**ShooterGame Reference Project:** 
+
+As you port your own project, you may find it helpful to use our ported [Unreal ShooterGame](https://docs.unrealengine.com/en-us/Resources/SampleGames/ShooterGame) running on the GDK as a reference.
```

Comment:
```suggestion
As you port your own Unreal project to SpatialOS, you could use our pre-ported Unreal Shooter Game as a reference.
```
"running on" suggests it is already running (as in running right now). Also, if we are saying it is pre-ported, we don't need to say it runs on the GDK, I think?
Also, we probs don't want to link to it here as we say below that you probs already have it. So we send people off to get it and then say "Oh, by the way, you already have it, probs. LOL! Trolled you!"